jetty-maven-plugin/pom.xml

```diff
@@ -252,120 +252,6 @@
       <artifactId>jetty-slf4j-impl</artifactId>
       <scope>test</scope>
     </dependency>
-    <dependency>
```

Comment:
nope it depends on it. 
see comment here https://github.com/eclipse/jetty.project/pull/8898#discussion_r1024496941

Reply:
Yet the build is green without it present.

Reply:
With this present, we get ...

```
[ERROR] [ERROR] The projects in the reactor contain a cyclic reference: Edge between
   'Vertex{label='org.eclipse.jetty:javadoc:10.0.13-SNAPSHOT'}' and 
   'Vertex{label='org.eclipse.jetty:jetty-maven-plugin:10.0.13-SNAPSHOT'}' introduces to cycle in the graph
   org.eclipse.jetty:jetty-maven-plugin:10.0.13-SNAPSHOT --> 
   org.eclipse.jetty:jetty-home:10.0.13-SNAPSHOT --> 
   org.eclipse.jetty:javadoc:10.0.13-SNAPSHOT --> 
   org.eclipse.jetty:jetty-maven-plugin:10.0.13-SNAPSHOT @ 
[ERROR] The projects in the reactor contain a cyclic reference: Edge between 
   'Vertex{label='org.eclipse.jetty:javadoc:10.0.13-SNAPSHOT'}' and 
   'Vertex{label='org.eclipse.jetty:jetty-maven-plugin:10.0.13-SNAPSHOT'}' introduces to cycle in the graph
   org.eclipse.jetty:jetty-maven-plugin:10.0.13-SNAPSHOT --> 
   org.eclipse.jetty:jetty-home:10.0.13-SNAPSHOT --> 
   org.eclipse.jetty:javadoc:10.0.13-SNAPSHOT --> 
   org.eclipse.jetty:jetty-maven-plugin:10.0.13-SNAPSHOT -> [Help 1]
```

Perhaps we should remove jetty-maven-plugin from the javadoc?